src/app/components/CourseChoice/CourseChoice.module.css

```diff
@@ -1,9 +1,9 @@
 .courseChoice {
   display: grid;
-  grid-template-columns: 1fr 3fr 1fr;
+  grid-template-columns: auto 1fr;
   align-items: center;
   column-gap: 0.5em;
-  width: 80%;
+  width: 100%;
   margin-left: auto;
```

Comment:
You could put these two together as `margin: 0 auto` :)

Reply:
Thank you! I could delete this at all. I made the alignment on the page!